waiter/src/waiter/service_description.clj

```diff
@@ -78,6 +78,7 @@
    (s/optional-key "namespace") schema/non-empty-string
    (s/optional-key "permitted-user") schema/non-empty-string
    (s/optional-key "ports") schema/valid-number-of-ports
+   (s/optional-key "profile") schema/non-empty-string
```

Comment:
Should this be a all lowercase no hyphen etc etc string?

Reply:
It is just a name, there is a separate check that validates whether the name is supported, i.e. matches something from the config.